src/twisted/web/newsfragments/.gitignore

```diff
@@ -0,0 +1 @@
+!.gitignore
```

Comment:
But it doesn't exist for git, it exists for towncrier.  (I don't even know what a `!` means in this file, and if I saw these lying around without this context, I'd delete them because I would have assumed some tool mistakenly created them.)